azext_iot/_params.py

```diff
@@ -412,6 +412,9 @@ def load_arguments(self, _):
         context.argument('repair', options_list=['--repair', '-r'],
                          arg_type=get_three_state_flag(),
                          help='Reinstall uamqp dependency compatible with extension version. Default: false')
+        context.argument('central_api_uri', options_list=['--central-api-uri'],
+                         help='IoT Central API override.  For use with environments other than production. '
+                         'Default: api.azureiotcentral.com.')
```

Comment:
Are you sure you need to add the 'Default: api.azureiotcentral.com' part of the string? The knack arg parser should automatically add the default value if its assigned to an arg in the command bound function. Please double check.

Reply:
The other explicit 'Default: blah' are for some limited cases *usually* for the 'three state flag' args i.e. when you require switches/flags.

Reply:
Yep you are right, I have removed and tested it.  Thanks!